packages/loopring_v3/contracts/lib/EIP712.sol

```diff
@@ -0,0 +1,70 @@
+/*
+
+  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless _requirement by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+pragma solidity ^0.5.11;
```

Comment:
This would certainly fix one of two problems we have with the current compiler. I'm not sure what the security implications could be by doing the upgrade. But maybe now is a good idea to upgrade as the smart contracts won't be used on mainnet for some time.